documentation/htc_workloads/workload_planning/htcondor_job_submission.md

```diff
@@ -0,0 +1,197 @@
+* The **Open Science Pool (OSPool)** is the set of resources your job runs on. It is composed of execution points, as well as other technologies, that compose the cpus, memory, and disk space that will run the computations of your jobs. 
+
+## Run Jobs on the OSPool using HTCondor
+We are going to run the traditional 'hello world' program with a OSPool twist. In order to demonstrate the distributed resource nature of OSPool HTC System, we will produce a 'Hello CHTC' message 3 times, where each message is produced within is its own 'job'. Since you will not run execution commands yourself (HTCondor will do it for you), you need to tell HTCondor how to run the jobs for you in the form of a submit file, which describes the set of jobs.
+
+> Note: You must be logged into an OSPool Access Point for the following example to work.
+
+## Prepare job executable and submit file on an Access Point
+1. First, create the executable script you would like HTCondor to run.  For our example, copy the text below and paste it into a file called `hello-ospool.sh` (we recommend using a command line text editor) in your home directory.
+
+```
+#!/bin/bash
+#
+# hello-ospool.sh
+# My very first OSPool job
+#
+# print a 'hello' message to the job's terminal output:
+echo "Hello OSPool from Job $1 running on `whoami`@`hostname`"
+#
+# keep this job running for a few minutes so you'll see it in the queue:
+sleep 180
+```
+
+This script would be run locally on our terminal by typing `hello-ospool.sh <FirstArgument>`. However, to run it on the OSPool, we will use our HTCondor submit file to run the `hello-ospool.sh` executable and to automatically pass different arguments to our script. 
+
+
+2. Prepare your HTCondor submit file, which you will use to tell HTCondor what job to run and how to run it. Copy the text below, and paste it into file called `hello-ospool.sub`. This is the file you will submit to HTCondor to describe your jobs (known as the submit file).
+
+```
+# hello-ospool.sub
+# My very first HTCondor submit file
+
+# Specify your executable (single binary or a script that runs several
+#  commands) and arguments to be passed to jobs. 
+#  $(Process) will be a integer number for each job, starting with "0"
+#  and increasing for the relevant number of jobs.
+executable = hello-ospool.sh
+arguments = $(Process)
+
+# Specify the name of the log, standard error, and standard output (or "screen output") files. Wherever you see $(Cluster), HTCondor will insert the 
+#  queue number assigned to this set of jobs at the time of submission.
+
+log = hello-ospool_$(Cluster)_$(Process).log
+error = hello-ospool_$(Cluster)_$(Process).err
+output = hello-ospool_$(Cluster)_$(Process).out
+
+# Specify that HTCondor should transfer files to and from the
+#  computer where each job runs. The last of these lines *would* be
+#  used if there were any other files needed for the executable to use.
+should_transfer_files = YES
```

Comment:
```suggestion
```

Reply:
I'm leaving this because it is required for certain Jupyter workflows.